src/sage/topology/simplicial_complex.py

```diff
@@ -1938,8 +1939,8 @@ def suspension(self, n=1, is_mutable=True):
             sage: S0 = SimplicialComplex([[0], [1]])
             sage: S0.suspension() == simplicial_complexes.Sphere(1)
             True
-            sage: S3 = S0.suspension(3)  # the 3-sphere
-            sage: S3.homology()
+            sage: S3 = S0.suspension(3)  # the 3-sphere                                 # optional - sage.graphs
+            sage: S3.homology()                                                         # optional - sage.graphs
             {0: 0, 1: 0, 2: 0, 3: Z}
```

Comment:
No `sage.modules`?

Reply:
Fixed

Reply:
Or are these homology groups? Or trivial `0`, `Z` does not need groups nor modules?

Reply:
That's possible, but I think it's fine to mark it `# optional - sage.modules` nevertheless. Easier to understand the pattern "whenever homology is computed, needs `# optional - sage.modules`.

Reply:
Yes.